CONTRIBUTING.md

```diff
@@ -113,12 +113,26 @@ The following is an overview of what's required in order to add support to the a
 
 If you have access to make releases, the process is as follows:
 
-1. Update the version in `package.json` according to the scale of the change. (major, minor or patch) 
-1. Add commit messages to the changelog (You may skip non-user-visible changes)
+#### Current major
+
+1. Be sure you have checked out the `master` branch and have pulled latest chnages
+1. Update the version in `package.json` according to the scale of the change. (major, minor or patch)
+1. Add commit messages to `CHANGELOG.md` (You may skip non-user-visible changes)
 1. Commit changes with message `x.y.z` where `x.y.z` is the version in `package.json`
 1. Tag the commit with `git tag vx.y.x`, for example `git tag v1.2.3`
-1. Reset the current major branch (`1.x`, `2.x` etc) to point to the current master, e.g. `git branch -f 1.x master`
+1. Reset the latest major branch (`1.x`, `2.x` etc) to point to the current master, e.g. `git branch -f 3.x master`
 1. Run tests with `npm test`
 1. Push commits and tags upstream with `git push upstream master && git push upstream --tags` (and optionally to your own fork as well)
-1. Update 1.x branch on upstream with `git push upstream 1.x`
+1. Update the latest major branch on upstream with `git push upstream <major_branch>`
+1. Publish to npm with `npm publish`
+
+#### Past major
+
+1. Be sure you have checked out the branch associated with the major you wish to release and have pulled latest chnages, e.g. `2.x`
+1. Update the version in `package.json` according to the scale of the change. (major, minor or patch)
+1. Add commit messages to `CHANGELOG.md` (You may skip non-user-visible changes)
```

Comment:
This will only update the `CHANGELOG.md` file in the past-major branch, and not in `master`. So when people go to view the changelog to see changes released to a past major, they will have to switch to that branch as well. This isn't that user-friendly... any recommendations?

Reply:
In APM Server we cherry pick these changelog additions to master.

Reply:
Do you then require the changelog changes to be in a separate commit? We normally include them in the commit that also bumps the version number, but we could of course change that behavior

Reply:
Not typically. I think they might just copy/paste the changes in. Example: https://github.com/elastic/apm-server/commit/5486d7b6974589639996c19b588b0c9d8b530297#diff-4c3654c19ee58cc14139418e0718783a. Also worth nothing is that APM Server has a "changelog check" (makes sure the changelogs remain in sync across branches) https://github.com/elastic/apm-server/blob/master/script/check_changelogs.py

Reply:
Thanks. Let's just agree to manually keep the branches in sync as you described and then possibly add something similar to `check_changelogs.py` as well.

Reply:
Works for me!